app/code/community/Reverb/ReverbSync/Model/Mapper/Product.php

```diff
@@ -106,6 +106,14 @@ public function getProductPrice($product)
         return $product->getPrice();
     }
 
+    public function getProductDescription($product)
+    {
+        $_reverb_allowed_tags = '<ul>,<ol>,<li>,<strong>,<b>,<em>,<i>,<u>,<div>,<p>,<br>';
+        $_descr = Mage::helper('core')->stripTags($this->getProductValueForListing($product, 'description'), $_reverb_allowed_tags);
```

Comment:
nevermind, as discussed in chat - you are setting the mapping automatically as a default.

curious why you're not using the getMagentoProductAttributeForReverbField function here?